# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/vulkan-headers/hotfix-no-modules.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- Vulkan-Headers-1.3.289/CMakeLists.txt.vanilla	2024-07-24 15:51:52.004243043 +0200
+++ Vulkan-Headers-1.3.289/CMakeLists.txt	2024-07-24 15:52:14.691240857 +0200
@@ -45,7 +45,7 @@
 if (MSVC AND (MSVC_VERSION GREATER_EQUAL "1934") OR
     CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "16.0" OR
     CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "14.0")
-    set(COMPILER_SUPPORTS_CXX_MODULES TRUE)
+    #set(COMPILER_SUPPORTS_CXX_MODULES TRUE)
 endif()
 
 option(VULKAN_HEADERS_ENABLE_MODULE "Enables building of the Vulkan C++ module. Default is true if supported by the CMake version and compilers" ${COMPILER_SUPPORTS_CXX_MODULES})
